### ISSUE TITLE
citrix_adc_nitro_request with count - missing 1 required positional argument: 'success_status' (Traceback)

### ISSUE THREAD

**Issue by Belabacsi** [citrix/citrix-adc-ansible-modules#221]
Dear Developers!
Dear GitHub Community!

I would like to ask for a little support regarding the **citrix_adc_nitro_request** Ansible module when **operation: count** is used.
I have been trying to run the following code with Ansible:

```
      - name: "Count"
        citrix.adc.citrix_adc_nitro_request:

            nsip: "{{ nsip }}"
            nitro_protocol: "{{ nitro_protocol }}"
            validate_certs: "{{ validate_certs }}"
            nitro_user: "{{ nitro_username }}"
            nitro_pass: "{{ nitro_password }}"

            operation: count
            resource: server

        register:  count_nsapp_lb_server_result

```

Unfortunately, I'm getting the following error message: 

`TypeError: edit_response_data() missing 1 required positional argument: 'success_status'`

Please find below my environment details:

```
$ ansible --version
ansible [core 2.12.4]
  config file = /home/ansible/.ansible.cfg
  configured module search path = ['/home/ansible/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/ansible/ansible/lib/python3.9/site-packages/ansible
  ansible collection location = /home/ansible/.ansible/collections:/usr/share/ansible/collections
  executable location = /home/ansible/ansible/bin/ansible
  python version = 3.9.7 (default, Sep 21 2021, 00:13:39) [GCC 8.5.0 20210514 (Red Hat 8.5.0-3)]
  jinja version = 3.0.3
  libyaml = True
```

Does anybody have the same experience with this? 

Thank You very much in advance for the support and help!

**Comment by vadharm** [comment]
@giorgos-nikolopoulos - Pls look into this.

**Comment by giorgos-nikolopoulos** [comment]
Fixed with #230
